### DISCUSSION TITLE
Did feed sharing just get disabled?

### DISCUSSION THREAD

**Discussion by CookieyedCodes** [listenbox/community#16] [Q&A] [answered]
Or did a person drop the subscription?
-a few of the big DnD channels feeds from LB are no longer updating, just wondering is it a policy change that be all :)

**Comment by meoyawn** [comment, marked as answer]
Feed sharing is working, but updating requires Listener/Creator subscription. I guess somebody canceled their subscription

**Comment by meoyawn** [comment]
no more feed sharing https://listenbox.app/

**Reply by CookieyedCodes** [reply]
Fair enough any plans for a free plan?

**Reply by meoyawn** [reply]
@CookieyedCodes we're serving Creators only now, no free plans https://listenbox.app/pricing/

**Reply by CookieyedCodes** [reply]
Fair enough, happy your back tho the space is more crowded now but I wish you luck getting creators on board
